pkg/utils/integration/space_requests.go

```diff
@@ -0,0 +1,26 @@
+package integration
```

Comment:
Technically, this resource is created by Gitops service ([here](https://github.com/redhat-appstudio/managed-gitops/blob/ac1312cc4eea936ea9ebe4845b9a48ba5c9d0474/appstudio-controller/controllers/appstudio.redhat.com/sandboxprovisioner_controller.go#L118)). I propose moving it under the "gitops" package.

Reply:
Moved  to "gitops" package.